### ISSUE TITLE
[Bug] nvim night build warning/error when starting

### ISSUE THREAD

**Issue by samarulmeu** [sindrets/diffview.nvim#493]
### Description

After upgrading to nvim ` v0.11.0-dev-6+g06135cc21` I get this warning/error

```
vim.tbl_add_reverse_lookup is deprecated. :help deprecated
Feature will be removed in Nvim 0.12
stack traceback:
        vim/shared.lua: in function 'tbl_add_reverse_lookup'                                                                                                                                                              ...e/username/.vim/plugged/diffview.nvim/lua/diffview/oop.lua:13: in function 'enum'
        ...name/.vim/plugged/diffview.nvim/lua/diffview/logger.lua:101: in main chunk                                                                                                                                    [C]: in function 'require'
        .../username/.vim/plugged/diffview.nvim/lua/diffview/lazy.lua:77: in function 'handler'
        .../username/.vim/plugged/diffview.nvim/lua/diffview/lazy.lua:23: in function ' get'                                                                                                                              .../username/.vim/plugged/diffview.nvim/lua/diffview/lazy.lua:44: in function 'Logger'
        ...ame/.vim/plugged/diffview.nvim/lua/diffview/bootstrap.lua:48: in main chunk                                                                                                                                     [C]: in function 'require'
        /home/username/.vim/plugged/diffview.nvim/plugin/diffview.lua:1: in main chunk                                                                                                                            stack traceback:
        ...ount_nvim.aOKQLza/usr/share/nvim/runtime/lua/vim/lsp.lua:1092: in function 'buf_get_clients'                                                                                                                   [string "luaeval()"]:1: in main chunk
```

Hope that helps,  thank you.

### Expected behavior

No error

### Actual behavior

No error on nvim start

### Steps to reproduce

Just start nvim

### Health check

<details>
<summary>Output of <code>:checkhealth diffview</code></summary>

```
diffview: require("diffview.health").check()

Checking plugin dependencies
- OK nvim-web-devicons installed.

Checking VCS tools
- The plugin requires at least one of the supported VCS tools to be valid.
- OK Git found.
- OK Git is up-to-date. (2.39.2)
- WARNING Configured hg_cmd is not executable: 'hg'
```

</details>

### Neovim version

```markdown
NVIM v0.11.0-dev-6+g06135cc21
Build type: RelWithDebInfo
LuaJIT 2.1.1713484068
```

### Operating system and version

Linux 6.1.0-21-amd64 x86_64 GNU/Linux

**Comment by fredrikaverpil** [comment]
@sindrets Neovim 0.10.0 was released. Would be awesome to get this fixed 😄

**Comment by sindrets** [comment]
Fixed by #492.
